pkg/apis/monitoring/v1/types.go

```diff
@@ -1828,6 +1828,9 @@ type RuleGroup struct {
 	// +kubebuilder:validation:Pattern="^(?i)(abort|warn)?$"
 	// +kubebuilder:default:=""
 	PartialResponseStrategy string `json:"partial_response_strategy,omitempty"`
+	// Limit the number of alerts an alerting rule and series a recording
+	// rule can produce.
```

Comment:
need to mention which Prometheus/Thanos versions are required

Reply:
Done